ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs

```diff
@@ -32,115 +29,172 @@
-                    Text = item.Title,
-                    LayoutParameters = ViewUtils.GetParameters(this)
-                };
-                container.AddView(child);
-                buttons.Add(child);
-            }
-        }
+                new ListItemButton(this, "Scan QR-/Barcode", ScanBarcode),
+                new ListItemButton(this, "Scan QR-/Barcode with image", ScanWithImageBarcode),
+                new ListItemButton(this, "Scan QR-/Barcode with selection overlay", ScanWithSelectionOverlayBarcode),
+                new ListItemButton(this, "Scan and Count Barcodes", ScanAndCountBarcodes),
+                new ListItemButton(this, "Scan Multiple QR-/Barcodes", ScanMultipleBarcodes),
+                new ListItemButton(this, "Scan MRZ", ScanMrz),
+                new ListItemButton(this, "Scan Generic Document", ScanGenericDocument),
+                new ListItemButton(this, "Scan data", ScanData),
+                new ListItemButton(this, "Scan VIN", ScanVIN),
+                new ListItemButton(this, "Scan EU license plate", ScanEULicensePlate),
+                new ListItemButton(this, "Scan European Health Insurance Card", ScanEhic),
+                new ListItemButton(this, "Medical Certificate recognizer", ScanMedicalCertificate),
+                new ListItemButton(this, "Check Recognizer", RecogniseCheck),
+            });
 
-        protected override void OnResume()
-        {
-            base.OnResume();
+            progress = FindViewById<ProgressBar>(Resource.Id.progressBar);
 
-            CheckLicense();
+            licenseIndicator = container.FindViewById<TextView>(Resource.Id.licenseIndicator);
+            licenseIndicator.Text = Texts.no_license_found_the_app_will_terminate_after_one_minute;
 
             foreach (var button in buttons)
             {
                 button.Click += OnButtonClick;
             }
         }
 
-        protected override void OnPause()
+        private void ScanAndCountBarcodes()
         {
-            base.OnPause();
-
-            foreach (var button in buttons)
-            {
-                button.Click -= OnButtonClick;
-            }
+            var configuration = new DocumentScannerConfiguration();
+
+            configuration.SetCameraPreviewMode(CameraPreviewMode.FitIn);
+            configuration.SetIgnoreBadAspectRatio(true);
+            configuration.SetMultiPageEnabled(true);
+            configuration.SetPageCounterButtonTitle("%d Page(s)");
+            configuration.SetTextHintOK("Don't move.\nScanning document...");
+
+            // further configuration properties
+            //configuration.SetBottomBarBackgroundColor(Color.Blue);
+            //configuration.SetBottomBarButtonsColor(Color.White);
+            //configuration.SetFlashButtonHidden(true);
+            // and so on...
+
+            //var intent = BarcodeScna.NewIntent(this, configuration);
+            //StartActivityForResult(intent, SCAN_DOCUMENT_REQUEST_CODE);
```

Comment:
@yurii-scanbot, is this intent and StartActivityForResult call needed? It doesn't look correct either in terms of BarcodeScna.